docs/scripts_md/tarchiveLoader.md

```diff
@@ -0,0 +1,93 @@
+# NAME
+
+tarchiveLoader.pl -- this script performs the following:
+
+\- validates the tarchive
+
+\- converts DICOM data into MINC files
+
+\- automated protocol checks against the entries in the \`mri\_protocol\` table.
```

Comment:
Maybe rephrase this section as below:

this script performs the following:
- validation of the tarchive
- conversion of DICOM datasets into MINC files
- automated protocol checks against entries in the mri_protocol and mri_protocol_checks tables